app/assets/javascripts/editorToolbar.js

```diff
@@ -22,6 +20,22 @@ $(function() {
     $E[action](); // call the appropriate editor function
   });
 
+  // for save & recover buttons
+  $('.save-button').on('click', function(e) {
+    $E.setState(e.currentTarget.dataset.formId); // string that is: "main", "reply-123", "edit-123" etc.
+    $E.save($E);
+  });
+
+  $('.recover-button').on('click', function(e) {
+    $E.setState(e.currentTarget.dataset.formId); // string that is: "main", "reply-123", "edit-123" etc.
+    $E.recover();
+  });
+
+  // textAreas
+  $('.text-input').on('click', function(e) {
+    $E.setState(e.currentTarget.dataset.formId);
```

Comment:
Video of `setState` when clicking into a comment form: https://user-images.githubusercontent.com/4361605/106988941-d9c5c580-6725-11eb-964c-6aa81728aeb7.mov